otter/test/test_cloud_client.py

```diff
@@ -719,6 +721,66 @@ def test_add_clb_nodes(self):
         # all the common failures
         self.assert_parses_common_clb_errors(expected.intent, eff)
 
+    def expected_node_removal_req(self, nodes=(1, 2)):
+        return service_request(
+            ServiceType.CLOUD_LOAD_BALANCERS,
+            'DELETE',
+            'loadbalancers/{}/nodes'.format(self.lb_id),
+            params={'id': map(str, nodes)},
+            success_pred=has_code(202))
+
+    def test_remove_clb_nodes_success(self):
+        """
+        A DELETE request is sent, and the Effect returns None if 202 is
+        returned.
+        """
+        eff = remove_clb_nodes(self.lb_id, [1, 2])
+        seq = [
+            (self.expected_node_removal_req().intent,
+             service_request_eqf(stub_pure_response({}, 202))),
+        ]
+        result = perform_sequence(seq, eff)
+        self.assertIs(result, None)
+
+    def test_remove_clb_nodes_non_202(self):
+        """Any random HTTP response code is bubbled up as an APIError."""
+        eff = remove_clb_nodes(self.lb_id, [1, 2])
+        seq = [
+            (self.expected_node_removal_req().intent,
+             service_request_eqf(stub_pure_response({}, 200))),
+        ]
+        self.assertRaises(APIError, perform_sequence, seq, eff)
+
+    def test_remove_clb_nodes_random_400(self):
+        """Random 400s are bubbled up as an APIError."""
+        eff = remove_clb_nodes(self.lb_id, [1, 2])
+        response = stub_pure_response(
+            {'validationErrors': {'messages': ['bar']}}, 400)
+        seq = [
+            (self.expected_node_removal_req().intent,
+             service_request_eqf(response)),
+        ]
+        self.assertRaises(APIError, perform_sequence, seq, eff)
```

Comment:
We should probably call `self.assert_parses_common_clb_errors(expected.intent, eff)` here or in the previous test.